src/legacy/app/actions/deviceActions.js

```diff
@@ -224,52 +143,76 @@ export function deselectDevice() {
     };
 }
 
+export const wrapDevice = device => {
+    const outputDevice = camelcaseKeys(device, { deep: true });
+    const serialport = outputDevice.serialPorts
+        ? outputDevice.serialPorts[0]
+        : undefined;
+    return {
+        ...outputDevice,
+        boardVersion: outputDevice.jlink
+            ? outputDevice.jlink.boardVersion
+            : undefined,
+        serialport,
+    };
+};
+
+export const wrapDevices = devices => devices.map(wrapDevice);
```

Comment:
Doesnt seem to be used externally.
```suggestion
const wrapDevices = devices => devices.map(wrapDevice);
```